### PR TITLE
Update `CA_Client_Say()` with new param: `bool: isTeamMessage`

### PR DESCRIPTION
**Pull request by JUNESYNGOTOFLEX**
compress to 1 forward

### CODE REVIEW

File: cstrike/addons/amxmodx/scripting/ChatAdditions_Core.sma [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#include <amxmodx>
```
Comment (Nord1cWarr1or) [inline cstrike/addons/amxmodx/scripting/ChatAdditions_Core.sma:1]: wtf

File: cstrike/addons/amxmodx/scripting/ChatAdditions_Core.sma [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+
```
Comment (wopox1337) [inline cstrike/addons/amxmodx/scripting/ChatAdditions_Core.sma:1]:
```suggestion
#include <amxmodx>
```

**Review by wopox1337 — CHANGES_REQUESTED** [review]

**Review by wopox1337 — CHANGES_REQUESTED** [review]
Edits are needed:
 - API;
 - using API components;
 - Minor version of the plugin.

File: cstrike/addons/amxmodx/scripting/ChatAdditions_Core.sma (hunk 2 above)
Comment (JUNESYNGOTOFLEX) [reply cstrike/addons/amxmodx/scripting/ChatAdditions_Core.sma]: from amxmiscinc

#if !defined _amxmodx_included
#include amxmodx
#endif

my opinion: dodge it xD

**Review by wopox1337 — APPROVED** [review]

**Review by wopox1337 — APPROVED** [review]

**Review by Nord1cWarr1or — APPROVED** [review]

**Pull request merged**